src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -681,7 +721,7 @@ private void handlePostPurchaseOrder(RoutingContext ctx) {
         .end(compPO.encodePrettily());
     }
 
-    private void handleGetVendor(RoutingContext ctx) {
+    private void handleGetOrganization(RoutingContext ctx) {
       logger.info("got vendor request: {}", ctx.request().query());
```

Comment:
Should this be `got organization request` because here organization can be vendor or non-vendor?

Reply:
Done